src/cli/log.js

```diff
@@ -0,0 +1,40 @@
+import tty from 'tty';
+import elegantSpinner from 'elegant-spinner';
+import logUpdate from 'log-update';
+import chalk from 'chalk';
+
+// NOTE: To support piping we use stderr as the
+// log output stream, while stdout is used
+// for the report output.
+export default {
+    animation: null,
+    isTTY:     tty.isatty(1),
+
+    showSpinner () {
+        // NOTE: we can use spinner only if stderr is a TTY,
+        // otherwise we can't delete previous animation frames
```

Comment:
```
        // NOTE: we can use the spinner only if stderr is a TTY,
        // otherwise we can't repaint animation frames
```
(if *repaint* is ok here)